2022-02-09 13:54:30,941 INFO    MainThread:482904 [wandb_setup.py:_flush():71] setting env: {}
2022-02-09 13:54:30,942 INFO    MainThread:482904 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-09 13:54:30,942 INFO    MainThread:482904 [wandb_init.py:_log_setup():371] Logging user logs to /home/stephen/Documents/3DDet/tools/wandb/run-20220209_135430-2oxdwwch/logs/debug.log
2022-02-09 13:54:30,942 INFO    MainThread:482904 [wandb_init.py:_log_setup():372] Logging internal logs to /home/stephen/Documents/3DDet/tools/wandb/run-20220209_135430-2oxdwwch/logs/debug-internal.log
2022-02-09 13:54:30,942 INFO    MainThread:482904 [wandb_init.py:init():404] calling init triggers
2022-02-09 13:54:30,942 INFO    MainThread:482904 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-02-09 13:54:30,942 INFO    MainThread:482904 [wandb_init.py:init():460] starting backend
2022-02-09 13:54:30,942 INFO    MainThread:482904 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-09 13:54:30,958 INFO    MainThread:482904 [backend.py:ensure_launched():216] starting backend process...
2022-02-09 13:54:30,967 INFO    MainThread:482904 [backend.py:ensure_launched():221] started backend process with pid: 482930
2022-02-09 13:54:30,969 INFO    MainThread:482904 [wandb_init.py:init():469] backend started and connected
2022-02-09 13:54:30,972 INFO    MainThread:482904 [wandb_init.py:init():533] updated telemetry
2022-02-09 13:54:30,993 INFO    MainThread:482904 [wandb_init.py:init():563] communicating current version
2022-02-09 13:54:31,620 INFO    MainThread:482904 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-09 13:54:31,620 INFO    MainThread:482904 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-09 13:54:31,874 INFO    MainThread:482904 [wandb_init.py:init():606] starting run threads in backend
2022-02-09 13:54:33,934 INFO    MainThread:482904 [wandb_run.py:_console_start():1810] atexit reg
2022-02-09 13:54:33,934 INFO    MainThread:482904 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-02-09 13:54:33,934 INFO    MainThread:482904 [wandb_run.py:_redirect():1689] Redirecting console.
2022-02-09 13:54:33,935 INFO    MainThread:482904 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-09 13:54:33,936 INFO    MainThread:482904 [wandb_init.py:init():633] run started, returning control to user process
2022-02-09 13:54:33,936 INFO    MainThread:482904 [wandb_run.py:_config_callback():956] config_cb None None {}
2022-02-09 13:54:33,937 INFO    MainThread:482904 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-02-09 13:54:33,939 INFO    MainThread:482904 [wandb_run.py:_restore():1752] restore
2022-02-09 13:54:36,445 INFO    MainThread:482904 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 751
  total_bytes: 751
}

2022-02-09 13:54:36,556 INFO    MainThread:482904 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 751
  total_bytes: 751
}

2022-02-09 13:54:36,745 INFO    MainThread:482904 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 751
  total_bytes: 751
}

2022-02-09 13:54:36,907 INFO    MainThread:482904 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 751
  total_bytes: 11521
}

2022-02-09 13:54:37,011 INFO    MainThread:482904 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 751
  total_bytes: 11521
}

2022-02-09 13:54:37,115 INFO    MainThread:482904 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 751
  total_bytes: 11521
}

2022-02-09 13:54:37,218 INFO    MainThread:482904 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 7620
  total_bytes: 11521
}

2022-02-09 13:54:37,320 INFO    MainThread:482904 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11521
  total_bytes: 11521
}

2022-02-09 13:54:37,423 INFO    MainThread:482904 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11521
  total_bytes: 11521
}

2022-02-09 13:54:37,525 INFO    MainThread:482904 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11521
  total_bytes: 11521
}

2022-02-09 13:54:37,627 INFO    MainThread:482904 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11521
  total_bytes: 11521
}

2022-02-09 13:54:37,937 INFO    MainThread:482904 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11521
  total_bytes: 11521
}

2022-02-09 13:54:38,371 INFO    MainThread:482904 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 11521
  total_bytes: 11521
}
local_info {
}

2022-02-09 13:54:39,499 INFO    MainThread:482904 [wandb_run.py:_append_files():2180] logging synced files
